### DISCUSSION TITLE
Increasing the bodyParser.json ({limit: '1mb'}) and   app.use(bodyParser.urlencoded({ extended: false, limit: '1mb' }))

### DISCUSSION THREAD

**Discussion by fernandodelucca** [requarks/wiki#3792] [Help / Questions]
What is the impact of increasing the bodyParser.json ({limit: '1mb'}) and app.use (bodyParser.urlencoded ({extended: false, limit: '1mb'})) in the server/master.js file to support larger documents ?

Can this be parameterized in config.yml?

**Comment by NGPixel** [comment]
No impact, other than allowing malicious users to send large payloads to be parsed. So just don't put some random large value.
